2022q2/kde.adoc

```diff
@@ -0,0 +1,55 @@
+=== KDE on FreeBSD
+
+Links: +
+link:https://freebsd.kde.org/[KDE FreeBSD] URL: link:https://freebsd.kde.org/[https://freebsd.kde.org/] +
+link:https://community.kde.org/FreeBSD[KDE Community FreeBSD] URL: link:https://community.kde.org/FreeBSD[https://community.kde.org/FreeBSD]
+
+Contact: Adriaan de Groot <kde@FreeBSD.org>
+
+The KDE on FreeBSD project packages the software from the KDE Community, along with dependencies and related software, for the FreeBSD ports tree.
+The software includes a full desktop environment called KDE Plasma (for both X11 and Wayland) and hundreds of applications that can be used on any FreeBSD machine.
+
+The KDE team (kde@) is part of desktop@ and x11@ as well, building the software stack to make FreeBSD beautiful and usable as a daily-driver graphics-based desktop machine. The notes below describe **mostly** ports for KDE, but also include items of import to the entire desktop stack.
+
+==== KDE Stack
+
+KDE Gear releases happen each quarter, KDE Plasma updates once a month, and KDE Frameworks have a new release each month as well. These (large) updates land shortly after their upstream release and are not listed separately.
+
+* *astro/kstars* latest release 3.5.9.
+* *deskutils/grantleetheme* got an entry in UPDATING because of some unusual changes to the installed structure of the port.
+* *deskutils/kalendar* joined the KDE Gear releases.
+* *devel/okteta* updates to the octal viewer and editor.
```

Comment:
Clarifying: do you mean "octal" (base-8) here, or "octet" (fancy name for 8-bit bytes)? For reference, in the 2022Q1 report, you called it "an editor and viewer for binary data" and its pkg-descr says "Numerical encodings: Hexadecimal, Decimal, Octal, Binary".